query/bridges_test.go

```diff
@@ -0,0 +1,67 @@
+package query_test
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/csv"
+	"github.com/influxdata/flux/execute/executetest"
+	"github.com/influxdata/influxdb/query"
+	"github.com/influxdata/influxdb/query/mock"
+)
+
+type failWriter struct {
+	Err error
+}
+
+// Write returns max(p, w.Take), w.Err,
```

Comment:
_nit_ this comment must refer to a member `Take` that you later removed.

Reply:
Yes, good catch -- I'll fix this comment and merge.